### PR TITLE
Simplify MsB and migrate data to TS db

### PR DESCRIPTION
**Pull request by amCap1712**
With the mbid mapper performing great, the importance of MSIDs has been reduced. MsB is also a part of LB now with no intention of exposing a user facing API. There is no reason for maintaining a separate MsB db now. Further, the structure of the database can be simplified.

The proposal is to move data from MsB DB into a `messybrainz` schema in Timescale cluster. The benefit is that we can directly join MsB data with listens and mappings if ever needed. 

Further, the fields in MsB can be toned down to just artist_name, track_name and release_name. This is the data we intend to use in the MBID mapper in future and is also the minimum useful metadata to show to users for features that use MSIDs. Since, the number of fields is fixed and only 3 we can do without a hash. We do a case insensitive text match for the recording data and only insert a recording if its not present in the table.

> **Note**
> 1. We cannot use a UNIQUE INDEX with ON CONFLICT DO NOTHING here because of historical duplicates in the table. 
> 2. artist_msid and release_msid have now been removed.

### CODE REVIEW

**Comment by amCap1712** [comment]
Did a demo run on wolf and seems to work fine.

**Review by mayhem — APPROVED** [review]
Looks good, only minor comments. Yay!

File: listenbrainz/messybrainz/tests/test_data.py [hunk 1]
```diff
@@ -44,61 +42,32 @@
...
-    def test_add_artist_credit(self):
-        with messybrainz.engine.connect() as connection:
-            artist_msid = data.add_artist_credit(connection, 'Kanye West')
-            self.assertEqual(artist_msid, data.get_artist_credit(connection, 'Kanye West'))
-
-    def test_add_release(self):
-        with messybrainz.engine.connect() as connection:
-            release_msid = data.add_release(connection, 'The College Dropout')
-            self.assertEqual(release_msid, data.get_release(connection, 'The College Dropout'))
+        with timescale.engine.connect() as connection:
```
[the remark this answers is not in the document]
Comment (amCap1712) [reply listenbrainz/messybrainz/tests/test_data.py:48]: IIUC, the tests you mention are there so no further action needed?

[the remark this answers is not in the document]
Comment (mayhem) [reply listenbrainz/messybrainz/tests/test_data.py]: Correct.

File: listenbrainz/messybrainz/transfer_to_timescale.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        FETCH NEXT 50000 ROWS ONLY
+    """
+    with messybrainz.engine.connect() as msb_conn:
+        results = msb_conn.execute(text(query), last_row_id=last_row_id)
+        return results.fetchall()
+
+
+def insert_data(values):
+    raw_conn = timescale.engine.raw_connection()
+    query = """
```
Comment (mayhem) [inline listenbrainz/messybrainz/transfer_to_timescale.py:44]: I love that this comment is SQL, not python. :)

**Review by mayhem — APPROVED** [review]
Looks good, all good with fixing tests in next PR!

File: listenbrainz/messybrainz/transfer_to_timescale.py (hunk 2 above)
Comment (amCap1712) [reply listenbrainz/messybrainz/transfer_to_timescale.py:63]: :)

**Pull request merged**